e2e/tests/examples/data-driven.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { config } from 'resources/workbench-config';
+import GoogleLoginPage from 'app/page/google-login';
+import { withPage } from 'libs/page-manager';
+
+describe('Login tests:', () => {
```

Comment:
Are these going to execute on Circle? Is this desired?

Reply:
Examples should be excluded and I will make the necessary changes before merge. It's not yet excluded to show tests are working in CircleCI presubmit job.